src/main/webapp/app/shared/card-wrapper/card-wrapper.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, input } from '@angular/core';
+import { TranslateDirective } from 'app/shared/language/translate.directive';
+
+@Component({
+    selector: 'jhi-card-wrapper',
+    templateUrl: './card-wrapper.component.html',
+    imports: [TranslateDirective],
+})
+export class CardWrapperComponent {
+    title = input.required<string>();
+    maxWidth = input<string>('50rem');
+    minHeight = input<string>('91%');
+}
```

Comment:
Can we explain the default values a but more? 

The cleanest solution would be adding a variable for each and adding javadoc above how we got the these values

Reply:
For me it's not clear right now where these values come from 😅